semperfiwebdesign/all-in-one-seo-pack
# Use doc title for preview snippet

Issue #2475

## Proposed changes

Use doc title in the preview snippet on the Edit page when the the title field in our meta box is blank.

## Types of changes

- Improves existing functionality

## Checklist


## Testing instructions

Install this PR, then navigate to a Post and play around with the doc title and our title field. If the title field in our meta box is blank, then the snippet should use the doc title. Else it should always use the title in our meta box. You'll also want to test this with the Classic Editor.
